main.go

```diff
@@ -26,6 +26,7 @@ import (
 	// to ensure that exec-entrypoint and run can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
+	zaplog "go.uber.org/zap"
```

Comment:
just zap

Reply:
we have imported `"sigs.k8s.io/controller-runtime/pkg/log/zap"` before.